ext/standard/tests/strings/sprintf_error.phpt

```diff
@@ -25,16 +25,40 @@ $arg1 = 'one';
 $arg2 = 'two';
 
 // with one argument less than expected
-var_dump( sprintf($format1) );
-var_dump( sprintf($format2,$arg1) );
-var_dump( sprintf($format3,$arg1,$arg2) );
+try {
+    var_dump( sprintf($format1) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    var_dump( sprintf($format2,$arg1) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    var_dump( sprintf($format3,$arg1,$arg2) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
 
 // with two argument less than expected
-var_dump( sprintf($format2) );
-var_dump( sprintf($format3,$arg1) );
+try {
+    var_dump( sprintf($format2) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    var_dump( sprintf($format3,$arg1) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
 
 // with three argument less than expected
-var_dump( sprintf($format3) );
+try {
+    var_dump( sprintf($format3) );
+} catch (\Error $e) {
```

Comment:
Please catch ``\ValueError`` for all these try-catch blocks when applying the other changes.